actiontext/test/unit/model_test.rb

```diff
@@ -2,7 +2,7 @@
 
 require "test_helper"
 
-class ActionText::ModelTest < ActiveSupport::TestCase
+class ActionText::ModelTest < ActiveRecord::TestCase
```

Comment:
Tests are failing now FYI:

`/rails/actiontext/test/unit/model_test.rb:5:in `<top (required)>': uninitialized constant ActiveRecord::TestCase (NameError)`

I couldn't figure out the right combination of `requires` to get this working (hence the copy-paste).